agents/timestream_aggregator/timestream_aggregator.py

```diff
@@ -305,10 +353,10 @@ def write_frames_to_file(self):
         for f in self.frames:
             # Do not record flowcontrol frames
             if f.type == core.G3FrameType.none:
-                self.log.warn("Received flow control frame with value {v}." +
+                self.log.warn("Received flow control frame with value {v}. " +
```

Comment:
Just flagging this as another place where frame type is used to determine whether a frame is flow control or not.

Reply:
Removed the type check here, https://github.com/simonsobs/socs/pull/50/commits/9d8b74c536ca5d70be11ab6bf86eb595fd0b5a18, and now check for the flow control key instead. Even though this extra check shouldn't be necessary, it's been helpful to have.